dbt/sagerx/dbt_project.yml

```diff
@@ -35,7 +35,7 @@ models:
       +materialized: view
     intermediate:
       +schema: sagerx
-      +materialized: view
+      +materialized: table
```

Comment:
most impactful change here

Reply:
I think the reason we maybe wouldn't want to make this change would be that the materialized table has no schedule for being refreshed, other than when we run the build_marts DAG which rebuilds all the child models for the mart.. I'm leaving a comment on the original issue discussion as to another possible solution